test/unit/host_test.rb

```diff
@@ -710,8 +710,6 @@ def teardown
       hostgroup = FactoryGirl.create(:hostgroup, :with_puppet_orchestration)
       h = FactoryGirl.create(:host, :managed, :with_environment, :hostgroup => hostgroup)
       Setting[:manage_puppetca] = true
-      assert h.puppet_proxy.present?
-      assert h.puppetca?
```

Comment:
Basically it was changed - now the user has the power of deciding whether he wants to inherit or not.